specs.tex

```diff
@@ -53,7 +53,7 @@ \subsection{Arena}
         \figref{fig:arena}.
   \item Markers will be placed on columns such that there is a \SI{120}{mm} 
         gap at the bottom.
-  \item The scoring zones are squares of sides \si{3}{m} positioned with the
+  \item The scoring zones are squares of sides \si{2.8}{m} positioned with the
```

Comment:
I think the new size (since there isn't a tolerance) should be `(6000mm - 370mm) / 2 == 2815mm`, as otherwise we're introducing another error.

Additionally I suggest that we introduce a tolerance here (±50mm would give us plenty of room for manoeuvre).

If we do introduce a tolerance, then we can probably keep the zone size as a round number.

Reply:
fixed in ca0bed703de08ceb8eff2c100f8de5276c30fdd5